include/eve/arch/arm/sve/sve_true.hpp

```diff
@@ -0,0 +1,28 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch.hpp>
+
+namespace eve::detail
+{
+
+// Returns clear sve_true for type
+// Calling svptrue_b8 is OK for most cases, where you OK with
+// 1s in insignificant bits but sometimes you are not.
+template<typename T>
+svbool_t
+sve_true()
```

Comment:
very much on the fence with respect to where we should use typed or non-typed bool. We'll see I guess.

Reply:
they have to be typed. Or the 1's won't be where they should in the internal representation.
I guess this detail function can also be used for the True() constant

Reply:
They don't always have to be typed. In some contexts, like in load, you can use the b8 one.

The question is: do we always use typed or do we use b8 when we can.
The answer is basically how many bool constants will we have.

Also also probably we will need one constant for each type involved regardless of anything, so using the appropriate one might save us having a b8